### ISSUE TITLE
Sass Syntax Error v1.0.0-beta.10

### ISSUE THREAD

**Issue by betocantu93** [miguelcobain/ember-paper#974]
I just updated my app to v1.0.0-beta.10 and I got a few SASS Compiler errors:

First I got this error while trying to `$ember serve` after doing `$ember i ember-paper@v1.0.0-beta.10` 

```
Could not find the default SASS implementation. Run the default blueprint:
   ember g ember-cli-sass
Or install an implementation such as "node-sass" and add an implementation option. For example:
   sassOptions: {implementation: require("node-sass")}
```

So I ran `$ember g ember-cli-sass`

Followed by `$ember serve`

And the output:

```
Sass Syntax Error (SassCompiler) in /app/styles/color-palette.scss:476:17

Error: NoSuchMethodError: method not found: 'call'
Receiver: Closure 'DZ'
Arguments: [Instance of 'fJ']
$colors-export: export('palettes', $colors);
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  tmp/sass_compiler-input_base_path-FAJcsTim.tmp/app/styles/color-palette.scss 476:17    @import
  tmp/sass_compiler-input_base_path-FAJcsTim.tmp/app/styles/_custom-variables.scss 24:9  @import
  tmp/sass_compiler-input_base_path-FAJcsTim.tmp/app/styles/app.scss 1:9                 root stylesheet
```

**Comment by betocantu93** [comment]
v1.0.0-beta.9 works with the color deprecations warnings

**Comment by miguelcobain** [comment]
@betocantu93 ember-cli-sass is in a bit of a mess at the moment.

tl;dr is that newer versions allow users to provide a sass implementation for it to use (typically dart sass or node sass). You're probably using an ember-cli-sass that has this feature.

It seems that v8.0.1 provides a default implementation, so it should just work. Version v7.2.0 should also work.

You can also use newer versions and provide a sass implementation. In short, just follow ember-cli-sass readme.

Please upgrade to ember-paper `v1.0.0-beta.11` to get rid of a deprecation message as well.

I'm closing this. Please reopen/comment if issue persists.

**Comment by ReasonableDeveloper** [comment]
I have the exact same issue. Neither v1.0.0-beta.11 nor 1.0.0-beta.12 fixed it for me.

**Comment by miguelcobain** [comment]
@ReasonableDeveloper please provide the ember-cli-sass version you're using and what ember-cli-sass options are you specifying in ember-cli-build.js file, if any.

**Comment by ReasonableDeveloper** [comment]
Running ember-cli-sass on version 8.0.1 using the default options.

**Comment by miguelcobain** [comment]
@ReasonableDeveloper did you `npm install --save-dev sass` like specified in ember-cli-sass docs?

**Comment by ReasonableDeveloper** [comment]
No but I did run `ember g ember-cli-sass` after the `Could not find the default SASS implementation. Run the default blueprint:` error which installed it for me.
`"sass": "^1.13.2"` is a devDependency of my project

**Comment by miguelcobain** [comment]
@ReasonableDeveloper seems like ember-cli-sass-variables-export isn't compatible with dart sass which is the default implementation.

Can you please try `npm install --save-dev node-sass` and then in your ember-cli-build.js file pass these options:

```js
var nodeSass = require('node-sass');

var app = new EmberApp({
  sassOptions: {
    implementation: nodeSass
  }
});
```

**Comment by cmaturana** [comment]
When I update the libraries, it deletes the file content and when I restore it to what it was before, it gives me an error in syntax: `Error: Undefined variable: "$base-color".
        on line 8 of tmp/sass_compiler-input_base_path-V0bDmemK.tmp/app/styles/ember-paper-link.scss
`

**Comment by patodevilla** [comment]
I am still having issues with 1.0.0-beta.13


`Sass Syntax Error (SassCompiler) in app/styles/default-theme.scss:103:53`

`Error: argument $color of red($color) must be a color
        on line 103 of tmp/sass_compiler-input_base_path-xZCQVOKd.tmp/app/styles/default-theme.scss, in function red
        from line 103 of tmp/sass_compiler-input_base_path-xZCQVOKd.tmp/app/styles/default-theme.scss, in mixin paper-theme-variables
        from line 117 of tmp/sass_compiler-input_base_path-xZCQVOKd.tmp/app/styles/default-theme.scss
        from line 64 of tmp/sass_compiler-input_base_path-xZCQVOKd.tmp/app/styles/ember-paper.scss
        from line 41 of tmp/sass_compiler-input_base_path-xZCQVOKd.tmp/app/styles/app.scss
 #{$paletteName}-#{$hue}: #{red($color)}, #{green($color)}, #{blue($color)};`

`Stack Trace and Error Report: /var/folders/sy/rkvfjlxx2_s7dpnhz47k1lq40000gn/T/error.dump.ad4b5020cad0e028aa7c7b2b1f9bfbeb.log`

**Comment by cmaturana** [comment]
> after installin node-sass, setting sassOptions and installing ember-cli-sass 7.2.0 I am still having issues with 1.0.0-beta.13
> 
> `Sass Syntax Error (SassCompiler) in app/styles/default-theme.scss:103:53`
> 
> `Error: argument $color of red($color) must be a color on line 103 of tmp/sass_compiler-input_base_path-xZCQVOKd.tmp/app/styles/default-theme.scss, in function red from line 103 of tmp/sass_compiler-input_base_path-xZCQVOKd.tmp/app/styles/default-theme.scss, in mixin paper-theme-variables from line 117 of tmp/sass_compiler-input_base_path-xZCQVOKd.tmp/app/styles/default-theme.scss from line 64 of tmp/sass_compiler-input_base_path-xZCQVOKd.tmp/app/styles/ember-paper.scss from line 41 of tmp/sass_compiler-input_base_path-xZCQVOKd.tmp/app/styles/app.scss #{$paletteName}-#{$hue}: #{red($color)}, #{green($color)}, #{blue($color)};`
> 
> `Stack Trace and Error Report: /var/folders/sy/rkvfjlxx2_s7dpnhz47k1lq40000gn/T/error.dump.ad4b5020cad0e028aa7c7b2b1f9bfbeb.log`

it's a very similar problem to me, finally i try install libraries but i lost the style of my app

**Comment by woohoou** [comment]
> `Sass Syntax Error (SassCompiler) in app/styles/default-theme.scss:103:53`

Same problem here.

**Comment by chbonser** [comment]
Should this issue be reopened?

I tried:
- beta.19 with ember-cli-sass 8.0.1 and sass 1.15.1. Syntax Error.
- beta.19 with ember-cli-sass 8.0.1 and node-sass installed and configured in ember-cli-build. Syntax Error.
